src/utils/useAtomsDevtools.ts

```diff
@@ -8,8 +8,13 @@ import { getReduxExtension } from './redux-extension/getReduxExtension';
 import { SnapshotOptions, useAtomsSnapshot } from './useAtomsSnapshot';
 import { useGotoAtomsSnapshot } from './useGotoAtomsSnapshot';
 
-const atomToPrintable = (atom: AnyAtom) =>
-  atom.debugLabel ? `${atom}:${atom.debugLabel}` : `${atom}`;
+const atomToPrintable = (atom: AnyAtom) => {
```

Comment:
Any thoughts on [updating the fallback to something like `<unlabeled-${atom}>`](https://github.com/jotaijs/jotai-devtools/blob/227170689eb68f006c5e60677d581f4f932f089d/src/DevTools/utils/atom-to-printable.ts#L11) 🤔. We do it for the UI version but not really for the Redux and React DevTools version

Reply:
I like it, but let's move this logic change / centralization to a separate PR.